packages/good-design/src/core/buttons/LearnButton.tsx

```diff
@@ -29,6 +29,11 @@ const linksNew: Record<learnSources, links> = {
     link: "https://gooddollar.notion.site/What-is-signing-b0019fe6c43241068050c9aa16e87ee1",
     label: "What is signing?",
     icon: BillySign
+  },
+  identity: {
+    link: "https://www.notion.so/gooddollar/Get-G-873391f31aee4a18ab5ad7fb7467acb3",
+    label: "Learn about the identification process",
+    icon: BillySign
```

Comment:
is this related?

Reply:
Not for the functioning of the microbridge, but to release to protocolUI.

(New modals where added a while ago, the identity one never got updated)